2-mandatory/3-financial-times.js

```diff
@@ -4,34 +4,62 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
-function potentialHeadlines(allArticleTitles) {
+function potentialHeadlines(ARTICLE_TITLES) {
     // TODO
+    let arr = [];
+    for (i = 0; i < ARTICLE_TITLES.length; i++) {
+        if (ARTICLE_TITLES[i].length <= 65) {
+            arr.push(ARTICLE_TITLES[i]);
+        } 
+    } return arr;
 }
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
-function titleWithFewestWords(allArticleTitles) {
+function titleWithFewestWords(articleTitles) {
     // TODO
+        let shortest = "";
+    let fewestWordsSoFar = Number.MAX_VALUE;
+    for (i = 0; i < articleTitles.length; i++) {
+        let wordCount = articleTitles[i].match(/(\w+)/g).length;
+        if (wordCount < fewestWordsSoFar) {
+            fewestWordsSoFar = wordCount;
+            shortest = articleTitles[i];
+        }
+    } return shortest;
```

Comment:
`split`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split

Reply:
you need to declare the variable `i` -> `let i = 0; ...`